### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by mikebattista**
# PR checklist

**Quick summary of changes**
Refreshed README.md for the new branding and project structure. Updated screenshots.

**Which issue does this PR relate to?**

**Applies to the following platforms:**

- [ ] WinUI
- [ ] WPF
- [ ] UWP

**Anything that requires particular review or attention?**

**Do all automated tests pass?**

**Have automated tests been added for new features?**

**If you've changed the UI:**
  - Be sure you are including screenshots to show the changes.
  - Be sure you have reviewed the [accessibility checklist](accessibility.md).

**If you've included a new template:**
  - Be sure you reviewed the [Template Verification Checklist](https://github.com/microsoft/TemplateStudio/wiki/Checklist:-Template-Verification).
  - Be sure it's included in the list on this [UWP](https://github.com/microsoft/TemplateStudio/blob/main/docs/UWP/getting-started-endusers.md) or [WPF](https://github.com/microsoft/TemplateStudio/blob/main/docs/WPF/getting-started-endusers.md) getting started doc.

**Have you raised issues for any needed follow-on work?**

**Have docs been updated?**

**If breaking changes or different ways of doing things have been introduced, have they been communicated widely?**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
 # Template Studio
 
-*Template Studio* (aka *TS*) is a Visual Studio 2019 Extension that accelerates the creation of new **Universal Windows Platform (UWP)**, **Windows Presentation Foundation (WPF .NET Core 3.1)** and **WinUI 3 in Desktop** apps using a wizard-based experience. The resulting project is well-formed, readable code that incorporates great development features while implementing proven patterns and best practices. Sprinkled throughout the generated code we have links to documentation to provide useful insights.
-*TS* supports creating UWP apps in C# or VB.Net, WPF (.NET Core 3.1) apps in C# and **WinUI 3 in Desktop** apps in C# or C++.
+Template Studio is a suite of Visual Studio 2022 extensions that accelerates the creation of new WinUI 3, WPF, and UWP apps using a wizard-based experience.
```
Comment (mrlacey) [inline README.md:3]:
```suggestion
Template Studio is a suite of Visual Studio 2022 extensions that accelerate the creation of new WinUI 3, WPF, and UWP apps using a wizard-based experience.
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 
-![Template Studio UWP screenshot](./docs/resources/getting-started/WTS%20-%20Project%20Type.png)
+* Template Studio for WinUI (C#)
+* Template Studio for WinUI (C++) (FUTURE)
+* Template Studio for WPF
+* Template Studio for UWP
 
-![Template Studio WPF screenshot](./docs/resources/getting-started/WTSProjectTypeWPF.png)
+Projects created with these extensions contain well-formed, readable code and incorporate latest development features while implementing proven patterns and leading practices. The generated code also includes links to documentation and TODO comments that provide useful insight and guidance for turning the generated projects into production applications.
```
Comment (mrlacey) [inline README.md:10]:
```suggestion
Projects created with these extensions contain well-formed, readable code and incorporate the latest development features while implementing proven patterns and leading practices. The generated code includes links to documentation and TODO comments that provide useful insight and guidance for turning the generated projects into production applications.
```

**Review by mrlacey — APPROVED** [review]
In the future, not including spaces in filenames that will be part of a URL is preferable. But not something I'd force a change over.

It might also be useful to include captions with the screenshots to improve the clarity of what they show. They're quite similar and so the difference may easily be missed.

**Comment by mikebattista** [comment]
I removed spaces and added captions.

**Pull request merged**